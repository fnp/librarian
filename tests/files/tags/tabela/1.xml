<tabela>
  <wiersz>
    <kol>a</kol>
    <kol>b</kol>
  </wiersz>
  <wiersz>
    <kol>c</kol>
    <kol>d</kol>
  </wiersz>
</tabela>

<tabela ramka="1">
  <wiersz>
    <kol>a</kol>
    <kol>b</kol>
  </wiersz>
  <wiersz>
    <kol>c</kol>
    <kol>d</kol>
  </wiersz>
</tabela>
