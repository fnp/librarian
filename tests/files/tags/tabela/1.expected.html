<div id="book-text">
<table>
<tr>
<td>a</td>
<td>b</td>
</tr>
<tr>
<td>c</td>
<td>d</td>
</tr>
</table>
<table class="border">
<tr>
<td>a</td>
<td>b</td>
</tr>
<tr>
<td>c</td>
<td>d</td>
</tr>
</table>
</div>
